### PR TITLE
Chassis clean

### PR DESCRIPTION
**Pull request by LucienMorey**

### CODE REVIEW

**Review by outsidermm — APPROVED** [review]
LGTM

File: components/chassis.py [hunk 1]
```diff
@@ -270,15 +269,15 @@ def setup(self) -> None:
...
+            Pose2d(0, 0, 0),
             stateStdDevs=(0.05, 0.05, 0.01),
             visionMeasurementStdDevs=(0.4, 0.4, math.inf),
         )
         self.field_obj = self.field.getObject("fused_pose")
         self.module_objs: list[wpilib.FieldObject2d] = []
         for idx, _module in enumerate(self.modules):
             self.module_objs.append(self.field.getObject("s_module_" + str(idx)))
-        self.set_pose(Pose2d(4, 3.5, Rotation2d.fromDegrees(180)))
+        self.set_pose(Pose2d(0, 0, 0))
```
Comment (auscompgeek) [inline components/chassis.py:280]: Oh, this would explain why we ended up inverting the drive motors :sweat_smile:

Could you flip `drive_reversed` back?

**Review by auscompgeek — APPROVED** [review]
LGTM with comment

File: components/chassis.py (hunk 1 above)
Comment (LucienMorey) [reply components/chassis.py:280]: Does this still need to be done @auscompgeek after we experimented , it seems like everything needs to be reversed or  that whole pipeline just isn't working for some reason

Comment (james-ward) [reply components/chassis.py:280]: I'm going to merge this as-is. I think we will need to come back and set the initial pose differently depending on what alliance we are on, so we'll need to do that later.

**Pull request merged**